modules/tinymce/src/core/test/ts/webdriver/delete/QuirksChromeTest.ts

```diff
@@ -0,0 +1,83 @@
+import { RealKeys } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Env from 'tinymce/core/api/Env';
+
+describe('webdriver.tinymce.core.delete.QuirksChromeTest', () => {
+  before(function () {
+    if (!Env.browser.isChromium() && !Env.browser.isSafari()) {
+      this.skip();
+    }
+  });
+
```

Comment:
Just because the fix is for specific browsers doesn't mean we should only test on those browsers. Based on your description I assume this behaviour works on firefox - we should make sure that continues to be the case.

`skip` should be used to avoid browsers with problems we don't want to deal with, not to write tests just for certain browsers. Every `skip` is tech debt we take on as a risk and probably should deal with one day.

```suggestion
```

Reply:
Done!